### ISSUE TITLE
Issue with motion controller position on fresh build

### ISSUE THREAD

**Issue by cafeoh** [ValveSoftware/steamvr_unreal_plugin#79]
Hi,

I've been trying to make packaged build work for a few days now on the marketplace version of the plugin, but couldn't (no custom inputs work even though they work fine in editor). So I pulled the 4.23 dev version (since I've seen a couple other issues that sound similar, hoping it was fixed).

But every version I try from github (dev 4.23, 4.24, rel 4.23) seems to have the same issue where the motion controller transform is completely wrong.

Here's a clip of what it looks like (I understand that it's a bit hard to understand how it relates to the actual controller rotation, but I'm making slow rotation movements. (if you think it's necessary I can film my hand side by side 🤷‍♂) 

https://puu.sh/Ev5q5/4099fe969a.mp4

Any clue on what causes this? Alternatively any clue on why the packaged version never works with my custom inputs, despite the input and bindings being correct (I might have to make a separate issue for this, but quite a few users on the discord have tried to help and none of them could find any issue in the way I was using it).

**Comment by 1runeberg** [comment]
Hi @cafeoh !

The hand appears to be swapping. Is this a build from the sample project we have in our wiki:
https://github.com/ValveSoftware/steamvr_unreal_plugin/wiki/sample/SteamVRInputPlugin.zip

If not, could you try building from dev-4.23 using Visual Studio 2017 (not VS2019) the sample project above and see if you have the same issue?

As for your custom inputs, check some notes I had on Issue #78 - specifically Item (1) with checking if you are using the packaged inputs within SteamVR or an edited one from a prior session:
https://github.com/ValveSoftware/steamvr_unreal_plugin/issues/78#issuecomment-544202220

**Comment by cafeoh** [comment]
Hello,

Sorry for the late reply. I managed to fix the tracking problem by downgrading VC++ (Apparently a recent bug in the compiler screwed up transform calculation, from what someone told me on the UE discord).

For the inputs not working, getting the develop-4.23 version seem to have fixed it, even though previously compiling it (even with the transform bug) did not seem to work.
I did make sure the bindings were set on default and weren't overriden by some earlier manipulation.

Since this seems to be fixed in the latest 4.23 version, and the other bug was unrelated to the plugin, I'll close the issue. 

Thanks a lot!
